### ISSUE TITLE
Devnet always bootstrapping

### ISSUE THREAD

**Issue by LesnyRumcajs** [ChainSafe/forest#3089]
**Issue summary**

The local devnet is constantly in bootstrap and not in follow mode. This must be investigated as we want to follow as much as possible.

**Other information and links**
```
forest  | 2023-06-28T10:15:19.034185Z  INFO forest_filecoin::chain_sync::chain_muxer: Local node is behind the network, starting BOOTSTRAP from LOCAL_HEAD = 0 -> NETWORK_HEAD = 9
forest  | 2023-06-28T10:15:19.036867Z  INFO forest_filecoin::chain_sync::tipset_syncer: Validating tipset: EPOCH = 1, N blocks = 1
forest  | 2023-06-28T10:15:19.050677Z  INFO forest_filecoin::chain_sync::tipset_syncer: Validating tipset: EPOCH = 2, N blocks = 1
forest  | [DEBUG] storage provder f01000 burning 0.0
forest  | [DEBUG] ProofValidationBatch was nil, quitting verification
forest  | 2023-06-28T10:15:20.945600Z  INFO forest_filecoin::chain_sync::tipset_syncer: Validating tipset: EPOCH = 3, N blocks = 1
forest  | [DEBUG] storage provder f01000 burning 0.0
forest  | [DEBUG] ProofValidationBatch was nil, quitting verification
forest  | 2023-06-28T10:15:20.947500Z  INFO forest_filecoin::chain_sync::tipset_syncer: Validating tipset: EPOCH = 4, N blocks = 1
forest  | [DEBUG] storage provder f01000 burning 0.0
forest  | [DEBUG] ProofValidationBatch was nil, quitting verification
forest  | 2023-06-28T10:15:20.949279Z  INFO forest_filecoin::chain_sync::tipset_syncer: Validating tipset: EPOCH = 5, N blocks = 1
forest  | [DEBUG] storage provder f01000 burning 0.0
forest  | [DEBUG] ProofValidationBatch was nil, quitting verification
forest  | 2023-06-28T10:15:20.951090Z  INFO forest_filecoin::chain_sync::tipset_syncer: Validating tipset: EPOCH = 6, N blocks = 1
forest  | [DEBUG] storage provder f01000 burning 0.0
forest  | [DEBUG] ProofValidationBatch was nil, quitting verification
forest  | 2023-06-28T10:15:20.952854Z  INFO forest_filecoin::chain_sync::tipset_syncer: Validating tipset: EPOCH = 7, N blocks = 1
forest  | [DEBUG] storage provder f01000 burning 0.0
forest  | [DEBUG] ProofValidationBatch was nil, quitting verification
forest  | 2023-06-28T10:15:20.954712Z  INFO forest_filecoin::chain_sync::tipset_syncer: Validating tipset: EPOCH = 8, N blocks = 1
forest  | [DEBUG] storage provder f01000 burning 0.0
forest  | [DEBUG] ProofValidationBatch was nil, quitting verification
forest  | 2023-06-28T10:15:20.956423Z  INFO forest_filecoin::chain_sync::tipset_syncer: Validating tipset: EPOCH = 9, N blocks = 1
forest  | [DEBUG] storage provder f01000 burning 0.0
forest  | [DEBUG] ProofValidationBatch was nil, quitting verification
forest  | 2023-06-28T10:15:20.957974Z  INFO forest_filecoin::chain_sync::chain_muxer: Bootstrap successfully completed, now evaluating the network head to ensure the node is in sync
forest  | 2023-06-28T10:15:20.957978Z  INFO forest_filecoin::chain_sync::chain_muxer: Evaluating network head...
forest  | 2023-06-28T10:15:39.034173Z  INFO forest_filecoin::chain_sync::chain_muxer: Local node is behind the network, starting BOOTSTRAP from LOCAL_HEAD = 9 -> NETWORK_HEAD = 14
forest  | 2023-06-28T10:15:39.034264Z  INFO forest_filecoin::chain_sync::tipset_syncer: Validating tipset: EPOCH = 9, N blocks = 1
forest  | 2023-06-28T10:15:39.034326Z  INFO forest_filecoin::chain_sync::tipset_syncer: Validating tipset: EPOCH = 10, N blocks = 1
forest  | [DEBUG] storage provder f01000 burning 0.0
forest  | [DEBUG] ProofValidationBatch was nil, quitting verification
forest  | 2023-06-28T10:15:39.036482Z  INFO forest_filecoin::chain_sync::tipset_syncer: Validating tipset: EPOCH = 11, N blocks = 1
forest  | [DEBUG] storage provder f01000 burning 0.0
forest  | [DEBUG] ProofValidationBatch was nil, quitting verification
forest  | 2023-06-28T10:15:39.038158Z  INFO forest_filecoin::chain_sync::tipset_syncer: Validating tipset: EPOCH = 12, N blocks = 1
forest  | [DEBUG] storage provder f01000 burning 0.0
forest  | [DEBUG] ProofValidationBatch was nil, quitting verification
forest  | 2023-06-28T10:15:39.039706Z  INFO forest_filecoin::chain_sync::tipset_syncer: Validating tipset: EPOCH = 13, N blocks = 1
forest  | [DEBUG] storage provder f01000 burning 0.0
forest  | [DEBUG] ProofValidationBatch was nil, quitting verification
forest  | 2023-06-28T10:15:39.041420Z  INFO forest_filecoin::chain_sync::tipset_syncer: Validating tipset: EPOCH = 14, N blocks = 1
forest  | [DEBUG] storage provder f01000 burning 0.0
forest  | [DEBUG] ProofValidationBatch was nil, quitting verification
forest  | 2023-06-28T10:15:39.042591Z  INFO forest_filecoin::chain_sync::chain_muxer: Bootstrap successfully completed, now evaluating the network head to ensure the node is in sync
forest  | 2023-06-28T10:15:39.042595Z  INFO forest_filecoin::chain_sync::chain_muxer: Evaluating network head...
forest  | 2023-06-28T10:15:59.035393Z  INFO forest_filecoin::chain_sync::chain_muxer: Local node is behind the network, starting BOOTSTRAP from LOCAL_HEAD = 14 -> NETWORK_HEAD = 19
forest  | 2023-06-28T10:15:59.035492Z  INFO forest_filecoin::chain_sync::tipset_syncer: Validating tipset: EPOCH = 14, N blocks = 1
forest  | 2023-06-28T10:15:59.035532Z  INFO forest_filecoin::chain_sync::tipset_syncer: Validating tipset: EPOCH = 15, N blocks = 1
forest  | [DEBUG] storage provder f01000 burning 0.0
forest  | [DEBUG] ProofValidationBatch was nil, quitting verification
forest  | 2023-06-28T10:15:59.037095Z  INFO forest_filecoin::chain_sync::tipset_syncer: Validating tipset: EPOCH = 16, N blocks = 1
forest  | [DEBUG] storage provder f01000 burning 0.0
forest  | [DEBUG] ProofValidationBatch was nil, quitting verification
forest  | 2023-06-28T10:15:59.038540Z  INFO forest_filecoin::chain_sync::tipset_syncer: Validating tipset: EPOCH = 17, N blocks = 1
forest  | [DEBUG] storage provder f01000 burning 0.0
forest  | [DEBUG] ProofValidationBatch was nil, quitting verification
forest  | 2023-06-28T10:15:59.039925Z  INFO forest_filecoin::chain_sync::tipset_syncer: Validating tipset: EPOCH = 18, N blocks = 1
forest  | [DEBUG] storage provder f01000 burning 0.0
forest  | [DEBUG] ProofValidationBatch was nil, quitting verification
forest  | 2023-06-28T10:15:59.041324Z  INFO forest_filecoin::chain_sync::tipset_syncer: Validating tipset: EPOCH = 19, N blocks = 1
forest  | [DEBUG] storage provder f01000 burning 0.0
forest  | [DEBUG] ProofValidationBatch was nil, quitting verification
forest  | 2023-06-28T10:15:59.042818Z  INFO forest_filecoin::chain_sync::chain_muxer: Bootstrap successfully completed, now evaluating the network head to ensure the node is in sync
forest  | 2023-06-28T10:15:59.042824Z  INFO forest_filecoin::chain_sync::chain_muxer: Evaluating network head...
```

**Comment by lemmih** [comment]
Should be fixed in #4328. Will close once this is confirmed.

**Comment by lemmih** [comment]
Fixed.
